libraries/content-tree-to-string/test.js

```diff
@@ -0,0 +1,53 @@
+import t from "tap"
+import {u} from "unist-builder"
+import stringify from "@content-tree/content-tree-to-string"
+import fs from "fs/promises"
+
+let testBase = "../../tests/content-tree-to-string"
+let inputNames = await fs.readdir(`${testBase}/input/`)
+
+for (let inputName of inputNames) {
+	let inputPath = `${testBase}/input/${inputName}`
+	let outputName = inputName.replace(/json$/, "text")
+	let outputPath = `${testBase}/output/${outputName}`
+	let inputText = await fs.readFile(inputPath, "utf8")
+	let input = JSON.parse(inputText)
+	try {
+		let output = await fs.readFile(outputPath, "utf8")
+		t.test(`${inputName} -> ${outputName}`, t => {
+			t.equal(stringify(input).trim(), output.trim())
```

Comment:
Do we care about whitespace for this function? This test would pass if stringify randomly had a load of trailing whitespace, maybe that doesn't matter?

Reply:
that is true ✅. we should probably put the trimming into the code itself. 

it was slightly hard to do so i just pretended it wasn't a problem instead